backend/routes/drops_intake.py

```
@@ -362,3 +398,67 @@ async def submit_to_intake(submission: IntakeSubmission):
     finally:
         if conn:
             await asyncio.to_thread(conn.close)
+
+
+@router.get("/pending")
+async def get_pending_drops(limit: int = 50, user=Depends(admin_required)):
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)